workflows/ampliseq.nf

```diff
@@ -40,6 +40,7 @@ if (params.qiime_ref_taxonomy && !params.skip_taxonomy && !params.classifier) {
  */
 
 single_end = params.pacbio ? true : params.single_end
+single_end = params.iontorrent ? true : params.single_end
```

Comment:
This doesn't work as intended. This line will overwrite `single_end = true` with `single_end = false` if `pacbio = true` and `iontorrent = false`. Thats why the PacBio test fails at the moment.

Reply:
Of course, thanks! Should be fixed now.